Paco/assets/custom_base.js

```diff
@@ -187,13 +187,23 @@ var paco = (function (init) {
       };
 
       function getLastEvent() {
-        return JSON.parse(window.db.getLastEvent());
+        return JSON.parse(window.db.getLastNEvents(num));
```

Comment:
Should this call the JavascriptEventLoader getLastEvent? That method takes no parameters. Otherwise, the num value either needs to be replaced with a 1.